tokio/src/runtime/threadpool/builder.rs

```diff
@@ -241,13 +240,6 @@ impl Builder {
         // Get a handle to the clock for the runtime.
         let clock = self.clock.clone();
 
-        // Get the current trace dispatcher.
-        // TODO(eliza): when `tokio-trace-core` is stable enough to take a
-        // public API dependency, we should allow users to set a custom
-        // subscriber for the runtime.
-        let dispatch = trace::dispatcher::get_default(trace::Dispatch::clone);
-        let trace = dispatch.clone();
```

Comment:
Hmm, I'm not convinced that this should be removed. While a single global default tracing subscriber can be set using `set_global_default`, there are still use cases that use scoped subscribers. 

Since tokio's runtime doesn't currently expose `around_worker` to the user, this integration is the only way users of the scoped subscriber API can propagate that subscriber to worker threads. 

WDYT about leaving the subscriber propagation in the runtime, but adding a feature flag for it? That flag could be off by default.

Reply:
I'm fine considering bringing it back later. This area of the code is going to churn a bunch and i would rather not carry over experimental parts. Once the core stabilizes, we can think about how to best integrate tracing again.

Reply:
Okay, I'm not going to block this PR over this, but I felt like it was worth bringing up.

Reply:
Okay, I'm not going to block this PR over this, but I felt like it was worth bringing up.